style.css

```diff
@@ -95,3 +98,71 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+.buttons {
+  display: flex;
+  gap: 50px;
+  margin: 60px;
+}
+
+.tooltip {
+  position: relative;
+}
+
+.tooltip-content {
+	position: absolute;
+  display: none;
+  top: -60px;
+  left: 0%;
+}
+
+.tooltip-inner-content {
+	position: relative;
+  padding: 10px;
+  color: white;
+  background-color: black;
+  border-radius: 8px;
+  white-space: nowrap;
+}
+
+#tooltip-1:hover + .tooltip-content {
```

Comment:
It is good and easy variant for this task. You can just try for yourself to play with mouse events to make same behaviour but by JS.